delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1070,6 +1083,9 @@ def loadBuildModules(buildModuleUrl){
 		environmentMetadataLoader = load "environment-deployment-metadata-loader.groovy"
 		echo "Environment metadata loader module loaded successfully."
 
+		apigeeModule = load "apigee-module.groovy"
```

Comment:
Can we load this module only when APIGEE integration is enabled by admin (when configLoader contains key: "APIGEE" - this can be a starting point)? We shouldn't load this module by default.

Reply:
Sure. I'll update that.

Ideally, we would move the switch-case logic to this part of the code, load the appropriate API gateway provider module, and then have a standardized call to the loaded module where the switch-case block is now.